tests/unit/ConversionNatureOfBusiness.spec.ts

```diff
@@ -117,9 +114,9 @@ describe('ConversionNatureOfBusiness without update', () => {
     await doneBtn.trigger('click')
```

Comment:
You click the button but you get an error, so you're still in edit mode (line 117).